pkg/action/action.go

```diff
@@ -362,7 +395,7 @@ func (cfg *Configuration) recordRelease(r *release.Release) {
 }
 
 // Init initializes the action configuration
-func (cfg *Configuration) Init(getter genericclioptions.RESTClientGetter, namespace, helmDriver string, log DebugLog) error {
+func (cfg *Configuration) Init(getter genericclioptions.RESTClientGetter, namespace, helmDriver string, log DebugLog, hookLogGetter HookLogGetter) error {
```

Comment:
Is the addition of a parameter here a breaking change?

Reply:
Can I still comment here after force-pushing?

Reply:
Yes, it is a breaking change. Programs using this function using the old function signature would no longer compile.

Reply:
We tried to find a way so that the logs aren't included in the output unless `--logs` is set. This is apparently not they way, since it breaks functionality, and I am pretty sure there is no other way to prohibit fetching the logs at all. What we could do is to either always include the logs (as before this recent change) or to remove the `logs` element from the object before printing it... Do you think any of that would be desirable?

Reply:
I think it's possible to have the cake and eat it in this case.

<img width="400" src="https://user-images.githubusercontent.com/7497533/149370800-0db6cca7-fc0e-45b6-aa3f-f3a812b46f8c.png" />

Reply:
There. I have also updated [the commit I created](https://github.com/simonalling/helm/commit/72ca6c485afcc4a6988506a672234b241770eae4) that only includes the `helm test` test suite, which I think should be merged as its own PR first. This PR can then be rebased/updated accordingly.